packages/office-ui-fabric-react/src/components/Callout/FocusTrapCallout.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+import { BaseComponent } from '../../Utilities';
+import { Callout } from './Callout';
+import { ICalloutProps } from './Callout.types';
+import { FocusTrapZone, IFocusTrapZoneProps } from '../FocusTrapZone';
+
+export interface IFocusTrapCalloutProps extends ICalloutProps {
```

Comment:
Shouldn't `IFocusTrapCalloutProps` in a FocusTrapCallout.types.ts file

Reply:
I guess it depends on whether it is truly necessary to export FocusTrapCallout in the index file